pkg/resmgr/control/cpu/cpu.go

```diff
@@ -128,13 +127,12 @@ func (ctl *cpuctl) enforceCpufreqGovernor(class string, cpusIds ...int) error {
 	if _, ok := ctl.classes[class]; !ok {
```

Comment:
nit: `ctl.classes[class]` is now evalutated 3 times. Once here, to bail out if class is unknown, once to check if there is a class-specific governor set (IOW it is != "") , and finally a 3rd time to get the actual value it is set to. It would be better to reorganize this function to evaluate both `ctl.classes[class]` and `ctl.classes[class].FreqGovernor` at most once.